### ISSUE TITLE
Using a Guix disk image directly

### ISSUE THREAD

**Issue by rekado** [giuliano108/guix-packages#1]
Hi,

I found your notes/Guix-on-WSL2.md very interesting.  Have you considered doing without busybox and using a Guix-generated disk image directly as the root file system?

If that works I'd be happy to see this added to the Guix Cookbook.

**Comment by giuliano108** [comment]
As far as I understand, you would need to use a Guix system to create the image/tarball. And if you don't have a Guix system, how would you bootstrap one? Unless you're suggesting someone takes on the task building the image, publishing the image on the Internet and keeping it up to date (I'm sure that's doable and would work)...

**Comment by rekado** [comment]
You don't need to run Guix System to use the "guix system" command.  You can generate disk images with "guix system disk-image" even on foreign distributions.

**Comment by giuliano108** [comment]
> You can generate disk images with "guix system disk-image" even on foreign distributions

Yes, you either need GuixSD or Guix running on a foreign distro. I tried to document a procedure that would allow someone to install Guix on WSL2 without either of those things. It's a lower "barrier to entry", in my opinion...
